app/controllers/course/assessment/question/controller.rb

```diff
@@ -25,6 +25,12 @@ def load_question_assessment_for(question)
     @assessment.question_assessments.find_by!(question: question.acting_as)
   end
 
+  def update_skill_ids_if_params_present
+    question_assessment_params = voice_response_question_params[:question_assessment]
```

Comment:
`voice_response_question_params` should have been updated too.. I will change it for you

Reply:
ah sorry, previously I accidentally force push the update from my local to here, so maybe you need to check if the current version is reverted back..

Though, I have refactored this one. So you might need to check on this again

Reply:
You have just overriden my commit...
I would suggest using force-with-lease instead of just force next time

Reply:
I'm sorry :( will not happen again next time. and sure noted on that.

Reply:
Dont worry :)